solidity-v1/dashboard/src/components/modal/coverage-pools/components/WithdrawalOverview.jsx

```diff
@@ -0,0 +1,209 @@
+    },
+  },
+  pending: {
+    label: {
+      color: colors.grey50,
+    },
+    tile: {
+      ...tileBaseStyle,
+      borderColor: colors.yellowSecondary,
+    },
+  },
+  new: {
+    label: {
+      color: colors.grey50,
+    },
+    tile: {
+      ...tileBaseStyle,
+      borderColor: colors.yellowSecondary,
+    },
+  },
+  progressBar: {
+    margin: "0",
+    marginTop: "0.5rem",
+  },
+  title: {
+    marginBottom: "0.5rem",
+  },
+}
+
+const statusToProgressBarColor = {
+  [PENDING_WITHDRAWAL_STATUS.COMPLETED]: colors.mint80,
+  [PENDING_WITHDRAWAL_STATUS.PENDING]: colors.yellowSecondary,
+  [PENDING_WITHDRAWAL_STATUS.EXPIRED]: colors.error,
+  [PENDING_WITHDRAWAL_STATUS.NEW]: colors.grey20,
+}
+
+const WithdrawOverviewTile = ({ title, amount, label, status }) => {
+  const { withdrawalDelay, withdrawalInitiatedTimestamp } =
+    useWithdrawalOverviewContext()
+
+  const getProgressBarProps = () => {
+    if (status === PENDING_WITHDRAWAL_STATUS.PENDING) {
+      const progressBarValueInSeconds = moment().diff(
+        moment.unix(withdrawalInitiatedTimestamp),
+        "seconds"
+      )
+      const progressBarTotalInSeconds = moment
+        .unix(withdrawalInitiatedTimestamp)
+        .add(withdrawalDelay, "seconds")
+        .diff(moment.unix(withdrawalInitiatedTimestamp), "seconds")
+      return {
+        total: progressBarTotalInSeconds,
+        value: progressBarValueInSeconds,
+        color: statusToProgressBarColor[status],
+      }
+    }
+
+    return {
+      total: 100,
+      value: 100,
+      color: statusToProgressBarColor[status],
+    }
+  }
+
+  return (
+    <div style={styles[status].tile}>
+      <h5 className="text-grey-50" style={styles.title}>
+        {title}
+      </h5>
+      <h4 className={"flex row center"}>
+        <TokenAmount
+          amount={amount}
+          token={covKEEP}
+          amountClassName={"text-small text-grey-70"}
+          symbolClassName={"text-small text-gray-70"}
+        />
+        <span className="text-small ml-a" style={styles[status].label}>
+          {label}
+        </span>
+      </h4>
+      <ProgressBar {...getProgressBarProps()} bgColor={colors.grey20}>
+        <ProgressBar.Inline height={10} style={styles.progressBar} />
+      </ProgressBar>
+    </div>
+  )
+}
+
+export const WithdrawalOverview = ({
+  existingWithdrawalCovAmount,
+  covAmountToAdd,
+  withdrawalDelay,
+  withdrawalInitiatedTimestamp,
+  withdrawalStatus,
+}) => {
+  // TODO get title based on the current status
+  const getTitle = () => {
+    if (withdrawalStatus === PENDING_WITHDRAWAL_STATUS.EXPIRED) {
+      return "expired withdrawal"
+    }
+    return "existing withdrawal"
+  }
+
+  const getLabel = () => {
+    if (withdrawalStatus === PENDING_WITHDRAWAL_STATUS.PENDING) {
+      return moment
+        .unix(withdrawalInitiatedTimestamp)
+        .add(withdrawalDelay, "seconds")
+        .format("MM/DD")
+    } else if (withdrawalDelay === PENDING_WITHDRAWAL_STATUS.COMPLETED) {
```

Comment:
Should be `withdrawalStatus` here

```suggestion
    } else if (withdrawalStatus === PENDING_WITHDRAWAL_STATUS.COMPLETED) {
```

Reply:
68f104f